tck/features/MergeNodeAcceptance.feature

```diff
@@ -48,7 +61,12 @@ Feature: MergeNodeAcceptance
 
   Scenario: Merge node with label add property on create
     Given an empty graph
-    When executing query: MERGE (a:Label) ON CREATE SET a.prop = 42 RETURN a.prop
+    When executing query:
+      """
+      MERGE (a:Label)
+        ON CREATE SET a.prop = 42
```

Comment:
There is no rule on this, but I saw it in one of the latest TCK files, and thought it a good idea. Do you reckon we don't do it, though?